### PR TITLE
add file tortiki.html

### PR DESCRIPTION
**Pull request by EkTartynova**
added changes to the structure of the site. Added Tortiki file.html

### CODE REVIEW

File: Tortiki.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE HTML>
+<html>
+ <head>
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+  <title>Пример веб-страницы_1</title>
+ </head>
+ <body>
+  <h1>Заголовок Tortiki.html</h1>
+  <!-- Комментарий -->
```
Comment (VitalyTartynov) [inline Tortiki.html]: Remove  comments

File: Tortiki.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    <p>Lorem ipsum dolor sit amet, <b>consectetur adipiscing elit</b>, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Lectus mauris ultrices eros in cursus turpis massa tincidunt. Ut eu sem integer vitae justo eget magna. Quis blandit turpis cursus in hac habitasse platea dictumst. At imperdiet dui accumsan sit amet nulla. Orci a scelerisque purus semper. Tristique senectus et netus et malesuada fames ac turpis egestas. Ut faucibus pulvinar elementum integer enim neque. Diam phasellus vestibulum lorem sed risus ultricies. Dui accumsan sit amet nulla facilisi morbi tempus iaculis. Nunc non blandit massa enim nec dui. Pellentesque elit ullamcorper dignissim cras tincidunt lobortis feugiat vivamus at. Diam phasellus vestibulum lorem sed risus ultricies tristique nulla. Donec et odio pellentesque diam volutpat commodo sed egestas egestas. Et tortor at risus viverra adipiscing at in tellus integer.</p>
+    <h1>Здесь новый заголовок</h1>
+    <p>Id diam maecenas ultricies mi. Diam quis enim lobortis scelerisque fermentum dui faucibus in. Vulputate odio ut enim blandit volutpat maecenas volutpat blandit aliquam. Vel fringilla est ul</p>   
+ </body>
+</html>
```
Comment (VitalyTartynov) [inline Tortiki.html]: Add new empty line in the end of file.

**Review by VitalyTartynov — CHANGES_REQUESTED** [review]
Fix some comments, please

File: Tortiki.html (hunk 1 above)
Comment (EkTartynova) [reply Tortiki.html]: comment removed

File: Tortiki.html (hunk 2 above)
Comment (EkTartynova) [reply Tortiki.html]: Blank string inserted

**Review by VitalyTartynov — APPROVED** [review]

**Pull request merged**